Sming/Arch/Esp8266/Compiler/ld/common.ld

```diff
@@ -173,11 +180,6 @@ SECTIONS
     /* debug_e() string pointers */
     *(.rodata._ZZ*log_string)
 
-    /* const references, mainly in templated code */
-    *(.rodata._ZN*)
-
-	*(.rodata._ZZN*)
-
```

Comment:
With the old toolchain this saved quite a bit of RAM, but with the new compiler it breaks because it also catches regular const strings. This feels generally unsafe so I've removed them. Will add some better ones.